src/_shared/js/ads.js

```diff
@@ -15,3 +23,18 @@ export function addTrackingPixel(rootNode) {
     pixel.src = '[%Trackingpixel%]%%CACHEBUSTER%%';
     return write(() => rootNode.appendChild(pixel));
 }
+
+// Sets the 'href' of the glabs edition link to the correct URL.
+export function setEditionLink (edition, linkElement) {
```

Comment:
This whole procedure could very much be simplified to

```
linkElement.href = GLABS_EDITION[edition] || GLABS_EDITION.default;
```

provided you convert keys to their all-uppercase counterpart.